python/nav/portadmin/snmputils.py

```diff
@@ -395,10 +395,18 @@ def get_cisco_voice_vlans(self):
     def set_cisco_voice_vlan(self, interface, voice_vlan):
         """Should not be implemented on anything else than Cisco"""
         raise NotImplementedError
+        
+    def set_cisco_cdp(self, interface, action):
```

Comment:
This should rather be named `enable_cisco_cdp`, to keep in line with the wording of the later method `disable_cisco_cdp`.

Reply:
I agree, but the existing 'cisco_voice_vlan' functions follow the same naming: 'set_cisco_voice_vlan' and 'disable_cisco_voice_vlan'.

Reply:
But the meaning of `set_cisco_voice_vlan` is also slightly different: It doesn't just enable Cisco VLAN, it *sets* an actual VLAN value for Cisco Voice.

Reply:
I see what you mean. Just pushed a new commit.